web/src/components/RunDetailTrace/Visualization.tsx

```diff
@@ -44,8 +45,9 @@ const Visualization = ({runState, spans, type}: IProps) => {
     (event, {id}: Node) => {
       TraceDiagramAnalyticsService.onClickSpan(id);
       dispatch(selectSpan({spanId: id}));
+      openDrawer(event);
```

Comment:
Hey @cescoferraro do we really need the `event` here?

Reply:
the toggle drawer method calls e.preventPropagation.
that's why I passed the event here. do you think I should remove it?

Reply:
I don't think you need it here.